$NetBSD: patch-src_syncterm_syncterm.man.in,v 1.2 2025/03/05 18:21:22 vins Exp $

Do not embed DESTDIR in installed man paths, instead honoring
PKG_SYSCONFDIR.

--- src/syncterm/syncterm.man.in.orig	2025-02-11 18:23:24.000000000 +0000
+++ src/syncterm/syncterm.man.in
@@ -119,7 +119,7 @@ The curses and ANSI modes are not necess
 a traditional screen mode. As such, filler will be added around the terminal
 screen. For some display sizes, unpredictable results may occur.
 .SH FILES
-.IP \fI%%DESTDIR%%%%PREFIX%%/etc/syncterm.lst\fR
+.IP \fI%%SYSTEM_LIST_DIR%%/syncterm.lst\fR
 The system wide dialing directory.
 .IP \fIsyncterm.lst\fR
 The users dialing directory.
